webull.py

```diff
@@ -210,7 +210,7 @@ def place_order(self, stock='', price=0, action='BUY', type='LMT', enforce='GTC'
          response = requests.post('https://tradeapi.webulltrade.com/api/trade/order/' + self.account_id + '/placeStockOrder', json=data, headers=headers)
          result = response.json()
 
-         if result['success'] == True :
+         if result['success']:
```

Comment:
Yes! I forget that one, fixed now.

Reply:
I'd suggest using the get method on dictionaries to avoid a KeyError when the result is empty/malformed. 

>if result.get('success') and result.get('code') == '200' :

Reply:
@alexonab The problem with that approach it's that if the ```response.json()``` return a exception like `ValueError` this code will never be executed, what about create a wrapper? Like:

```python
import requests

req = requests.get('http://httpbin.org/get')


class WebullResult(dict):
    def __getitem__(self, key):
        return dict.__getitem__(self, key) if key in self else None


def wrapper_result(request):
    try:
        return WebullResult(request.json())
    except ValueError:
        return WebullResult()


result = wrapper_result(req)

print(result['success'])
```

Reply:
I don't believe the provided wrapper would resolve cases where request.json() returns an exception.  Without the wrapper, the function would exit at the exception and wouldn't evaluate if result['success'].

Reply:
@alexonab Why not wrapper not work with `request.json()` exceptions? What do you suggest instead? Cheers.

Reply:
I don't think the error handling for requests needs to be anything too creative. I would suggest using response.raise_for_status(). Using .get is also notable slower than checking if the key exists before using it, which for algo stock trading apps speed should be of some importance. 
If it makes sense to catch an HTTP exception to retry, then this API can handle that, otherwise the user of the api should handle the raised exception.
Instead of returning a boolean, I think it's more common to return back the whole response and let the caller piece out what they are after:
        if response.text != '':
            return response.json()
        return None

This all would be a much bigger error handling improvement to this code than what @Urahara has done.  These changes are still an improvement. And I'm okay with this patchset as it is if @tedchou12 is.